playbooks/common/openshift-node/config.yml

```diff
@@ -28,7 +28,7 @@
   tasks:
   - name: Create local temp directory for syncing certs
     local_action: command mktemp -d /tmp/openshift-ansible-XXXXXXX
-    register: mktemp
+    register: maketemp
```

Comment:
We can remove these two temporary directory plays entirely. I just tested and they are no longer used.